app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -1,148 +1,198 @@
 package org.sil.storyproducer.model
 
+import com.squareup.moshi.Json
 import org.sil.storyproducer.R
 import org.sil.storyproducer.controller.MainActivity
 import org.sil.storyproducer.controller.RegistrationActivity
-import org.sil.storyproducer.controller.export.CreateActivity
+import org.sil.storyproducer.controller.export.FinalizeActivity
 import org.sil.storyproducer.controller.export.ShareActivity
 import org.sil.storyproducer.controller.learn.LearnActivity
 import org.sil.storyproducer.controller.pager.PagerBaseActivity
 import org.sil.storyproducer.controller.remote.WholeStoryBackTranslationActivity
 
-
+/**
+ * PhaseType enum used to track current phase
+ *
+ * Annotate the enum so Moshi knows what values it’s looking for when deserializing

```

Comment:
I'm guessing that the Json annotations are used now for both serializing and deserializing?
Would the final line of the comment be correct if something like:
"@Json annotations set the keys used by Json serializing and deserializing. 
At this time these keys are a match for SP v3.0.4 and previous story.json files."